src/Tokenizer/Analyzer/CommentsAnalyzer.php

```diff
@@ -65,7 +65,7 @@ public function isHeaderComment(Tokens $tokens, int $index): bool
      *
      * @see https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#3-definitions
      */
-    public function isBeforeStructuralElement(Tokens $tokens, int $index): bool
+    public function isBeforeStructuralElement(Tokens $tokens, int $index, bool $allowReturnStatement = false): bool
```

Comment:
From here, the new parameter does not make sense - `isBeforeStructuralElement` is clear, parameter `allowReturnStatement` allows cheating. Let's not do that and limit `allow_before_return_statement` functionality to only `PhpdocToCommentFixer`.

Reply:
Extracted into `isBeforeReturn` method